Sources/SwiftCliCore/NestedArrayBuilder.swift

```diff
@@ -8,4 +8,35 @@ public class NestedArrayBuilder {
         let rowStitches = substringRowStitches.map {(String($0))}
         return(rowStitches)
     }
+
+    public func expandRow(row: [String]) throws ->  [String] {
+        return try row.flatMap { try getMultipleStitchInfo(stitch: $0) }
+    }
+
+    private func getMultipleStitchInfo(stitch: String) throws -> [String] {
+        let isRepeatingStitch = repeatingStitches.first(where: { stitch.starts(with: $0.name )})
+        if let isRepeatingStitch = isRepeatingStitch {
+            let stitchName = isRepeatingStitch.name
+            var clipStitch = stitch
+            clipStitch.removeAll(where: { stitchName.contains($0) })
+            let stitchNameSuffix = clipStitch
+            guard let repeatNumber = Int(stitchNameSuffix) else {
```

Comment:
Not a blocker -- something for pairing later after you merge -- let's talk about refactoring the names here and maybe making a little clearer.